src/components/mofo-footer/mofo-footer.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+export default React.createClass({
+  propTypes: {
+    footerLinks: React.PropTypes.arrayOf(React.PropTypes.shape({
+      iconType: React.PropTypes.oneOf([`info`, `email`, `github`, `chat`, `twitter`, `facebook`, `cc-license`, `code-of-conduct`, `cookies`, `legal`, `privacy`, `donate`,]).isRequired,
+      link: React.PropTypes.string.isRequired,
+      text: React.PropTypes.string.isRequired,
+      className: React.PropTypes.string
```

Comment:
I've seen a red donate footer icon on a MoFo site before. If we don't allow additional `classname` to be passed then on the site itself we will have to do the `n-th` child selector trick to highlight a particular icon... this doesn't seem ideal to me. Thoughts?